src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -82,12 +82,19 @@ prevent the user from collapsing the items.
 </AppNavigationItem>
 ```
 ### Editable element
-Add the prop `:editable=true` and an edit placeholder if you need it. By devault
+Add the prop `:editable=true` and an edit placeholder if you need it. By default
 the placeholder is the previous title of the element.
 
 ```
 <AppNavigationItem title="Editable Item" :editable="true"
-	editPlaceholder="your_placeholder_here" />
+	editPlaceholder="your_placeholder_here" icon="icon-folder" @update:title="function(value){alert(value)}" />
+```
+
+### New Item element
+Add the prop `:new-item=true`.
+
+```
+<AppNavigationItem title="New Item" icon="icon-add" :new-item="true" @new-item="function(value){alert(value)}" />
```

Comment:
I'm wondering if creating a dedicated component for it wouldn't be easier.
And let the dev implement the v-if/else behaviour. cc @ChristophWurst @juliushaertl

Reply:
I thought of it, but currently everything AppNavigationItem related is directly in the one component (Collapse, undo, editable,  children). So i just implemented it similar to the thing already there.

Reply:
I think it would be a good idea to move the input with a confirm and cancel button into a seperate component, to reduce code duplication.

Reply:
Agree

Reply:
I moved the input with a confirm and cancel button into a separate component. I have to do a bit of code cleanup (mostly in the scss), but i think this PR can be merged soon :+1: 

@skjnldsv Should the input with a confirm and cancel component only be used by the libary itself or should we add it as a feature to the docs ? And should it stay in the /AppNavigationItem/ directory ?

Reply:
> Should the input with a confirm and cancel component only be used by the libary itself or should we add it as a feature to the docs ?

Well, I think it should replace the AppNavigationItem in such cases.
Maybe it's cleaner to completely separate it?

> And should it stay in the /AppNavigationItem/ directory ?

I would say it's fine to be used as `AppNavigationItemNew` or something

Reply:
I understand why it might be good to completely separate it, but it uses many classes from AppNavigationItem. The Rest of the Code can be separated easily, but the style of AppNavigationItem is currently set to scoped. Either it remains one component or we separate it and remove the scoped from the style tag.
I personally prefer that a component has all the style it needs in its own style tag and doesn't depend on a different component, wich would not be the case if we separate. But if you think that would be better i can definitely move it into a dedicated component.

Reply:
cc @nextcloud/vuejs

Reply:
I will seperate it into a own component tomorrow.

Reply:
The seperation is done (sorry for the delay)